test_scripts/UpdateVideoStreamingCapabilities/001_11_GetCapabilities_caching_of_addVSC_nesting.lua

```diff
@@ -0,0 +1,57 @@
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description: SDL persists the videoStreamingCapability with additionalVideoStreamingCapabilities received from HMI
+--  in UI.GetCapabilities response in case additionalVideoStreamingCapabilities parameter contains
+--  nested additionalVideoStreamingCapabilities parameter
+--
+-- Preconditions:
+-- 1. HMICapabilitiesCacheFile is set in smartDeviceLink.ini
+-- 2. SDL and HMI are started
+--
+-- Sequence:
+-- 1. SDL requests UI.GetCapabilities()
```

Comment:
Please update the numeration of steps according to comment in the script above

Reply:
Done (see 5bf783e)